main.go

```diff
@@ -168,6 +170,13 @@ func tagCommand() *cli.Command {
 				Usage:       "external tag group configuration file path",
 				DefaultText: "/path/to/conf/file/ (.yml/.yaml extension)",
 			},
+			&cli.StringSliceFlag{
+				Name:        parsersArgs,
+				Aliases:     []string{"i"},
+				Usage:       "IAC types to tag",
+				Value:       cli.NewStringSlice(),
```

Comment:
@alexjurkiewicz Default value should be all the possible IACs. That should also fix your test

Reply:
Ah, I thought `DefaultText` was the default value. Should have read the docs! Fixed.